### DISCUSSION TITLE
Image links are doubled when transferred

### DISCUSSION THREAD

**Discussion by strombraaten** [ObsidianPublisher/obsidian-github-publisher#323] [Q&A — Plugin] [answered]
For some reason when I upload a file where I've added an image then the link ends up looking like this: `![[images/Simens-skjermbilder-07-05-2023-kl09.58 1.png|images/Simens-skjermbilder-07-05-2023-kl09.58 1.png]]`

As you can see it's the same link twice. Is that something I've done with the settings?

Couldn't find anything that adressed the same thing with images, that's why I'm asking. But I have updated to the latest plugin-version, but haven't tried that yet, as I had a hell of a lot merge conflicts already 😓

**Comment by Mara-Li** [comment]
You mean, the alias?

**Reply by strombraaten** [reply]
Yeah, sorry, I mean the alias. But I don't know how, or why, there's an alias (identical to the file name) being added to the images I upload.

Aaaah! Maybe it's because I've been messing around in the settings:
![Simens-skjermbilder-16-03-2024-kl10 19](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/72508979/cbe725e7-6a39-42b9-a77c-54db40008477)

I'm guessing I should just stick to the Obsidian structure, right? Don't know why I turned that off, but that would explain why the `images/`-part is being added to my image links. My images are indeed withintin `content/images`, but based off what I see in the default quartz repo, it doesn't seem like there a need to actually specify the folder structure that way. Which could be why my image links are broken in production

**Reply by Mara-Li** [reply, marked as answer]
Alias doesn't change link. Alias are always added to link to prevent error during build and be consistant with dataview conversion. Also it's a better thing for user as they now what image is.

**Reply by strombraaten** [reply]
Oooh, I see, I didn't know I could write an alias to an image link like that. That's for screenreaders right? Or in a more common situtation, if the image files just won't load, then you see the alias instead, I assume?

That's great. Everything works fine now 👍

**Reply by Mara-Li** [reply]
Screen-reader or using the alt to set some css format. For example, I always use `|+right` to float the image to the right!
